app/jobs/reports/authentication_report.rb

```diff
@@ -9,29 +9,30 @@ class AuthenticationReport < BaseReport
     def perform(report_date)
       return unless IdentityConfig.store.s3_reports_enabled
       self.report_date = report_date
+      message = "Report: #{REPORT_NAME} #{report_date}"
+      subject =  "Weekly Authentication Report - #{report_date}"
 
       report_configs.each do |report_hash|
-        issuers = report_hash['issuers']
-        emails = report_hash['emails']
-        name = report_hash['name']
-        data = report_maker(issuers).to_csv
+        tables = weekly_authentication_report_csv(report_hash['issuers'])
 
-        emails.each do |email|
-          ReportMailer.authentication_report(
+        report_hash['emails'].each do |email|
+          ReportMailer.tables_report(
             email:,
-            name:,
-            issuers:,
-            data:,
+            subject:,
+            message:,
+            tables:,
           ).deliver_now
         end
       end
     end
 
-    def report_maker(issuers)
+    private
+
+    def weekly_authentication_report_csv(issuers)
       Reporting::AuthenticationReport.new(
         issuers:,
         time_range: report_date.all_week,
-      )
+      ).to_csv
```

Comment:
one quick note -- the `ReportMailer` takes in an array of arrays, like basically the raw rows. This is returning a single string which is the CSV --- I'd consider adding an `as_csv` method to the report (similar to how we separate `as_json` (JSON literals like hashes, arrays) from `to_json` (JSON string))

Reply:
oh good catch thank you

Reply:
updated in https://github.com/18F/identity-idp/pull/9295/commits/2639c7a8cab7932106235d5e5a2632ba3427dfb3